### PR TITLE
help descriptions for better clarity

### PR DESCRIPTION
**Pull request by noborus**
- Improved command-line help and help descriptions for better clarity.
- Fixed typos in various files.

### CODE REVIEW

File: oviewer/input_search_test.go [hunk 1]
```diff
@@ -278,14 +278,14 @@ func Test_stripBackSlash(t *testing.T) {
 		want string
 	}{
 		{
-			name: "nonEsacpe",
+			name: "nonEsacape",
```
Comment (ccoVeille) [inline oviewer/input_search_test.go:281]: This no?

```suggestion
			name: "nonEscape",
```

Or is it intentional?

File: oviewer/input_search_test.go [hunk 2]
```diff
@@ -278,14 +278,14 @@ func Test_stripBackSlash(t *testing.T) {
...
 			args: args{
 				str: "\test",
 			},
 			want: "\test",
 		},
 		{
-			name: "nonEsacpe2",
+			name: "nonEsacape2",
```
Comment (ccoVeille) [inline oviewer/input_search_test.go:288]: Same as https://github.com/noborus/ov/pull/635#discussion_r1793426408

File: oviewer/keybind.go [hunk 3]
```diff
@@ -365,7 +365,7 @@ func (k KeyBind) String() string {
 	k.writeKeyBind(&b, actionLastSection, "last section")
 	k.writeKeyBind(&b, actionFollowSection, "follow section mode toggle")
 	k.writeKeyBind(&b, actionSectionNum, "number of section header lines")
-	k.writeKeyBind(&b, actionHideOther, "toggle hide other section")
+	k.writeKeyBind(&b, actionHideOther, "hide other section toggle")
```
Comment (ccoVeille) [inline oviewer/keybind.go:368]: It's unclear if you hide other section,or iftou hide a section named "other"

So maybe this


```suggestion
	k.writeKeyBind(&b, actionHideOther,`"hide "other" section toggle`)
```

**Review by ccoVeille — COMMENTED** [review]
A few minor remarks and suggestions

File: oviewer/input_search_test.go (hunk 1 above)
Comment (noborus) [reply oviewer/input_search_test.go:281]: Oops

File: oviewer/keybind.go (hunk 3 above)
Comment (noborus) [reply oviewer/keybind.go:368]: Thank you.

File: oviewer/input_search_test.go [hunk 4]
```diff
@@ -278,14 +278,14 @@ func Test_stripBackSlash(t *testing.T) {
 		want string
 	}{
 		{
-			name: "nonEsacpe",
+			name: "nonEscaape",
```
Comment (ccoVeille) [inline oviewer/input_search_test.go:281]: I cannot stop laughing, thanks @noborus 

Try this


```suggestion
			name: "nonEscape",
```

Comment (noborus) [reply oviewer/input_search_test.go:281]: ...sorry

**Review by ccoVeille — APPROVED** [review]

**Comment by noborus** [comment]
Thank you.

**Pull request merged**